papertrail/action.yml

```diff
@@ -0,0 +1,42 @@
+name: "Papertrail Report"
+description: "Generate report for authorized publication on distribution channels"
+inputs:
+  product_name:
+    description: "Name of product"
+    required: true
+  release_version:
+    description: "The release version"
+    required: false
+  filenames:
+    description: "Artifact filenames to include in the report, space-separated"
+    required: true
+  token:
+    description: "The GitHub token for the action"
+    required: true
```

Comment:
Can we use `default: ${{ github.token }}` here or does that not work?

Reply:
Not from within an action, "Note: This context property is set by the Actions runner, and is only available within the execution steps of a job."

Reply:
I see. No problem then 👍